packages/connect/package.json

```diff
@@ -79,7 +79,8 @@
     "prepublishOnly": "yarn build",
     "start": "tsdx watch",
     "test": "tsdx test",
-    "typecheck": "tsc --noEmit"
+    "typecheck": "tsc --noEmit",
+    "version": "yarn"
```

Comment:
This is a workaround to [this bug](https://github.com/lerna/lerna/issues/1171) in Lerna.

We need to add this to every package that has a dependency inside of our monorepo. So, we need it in `app` and `connect`, but not `keychain` or `ui`.